manager/state/raft/raft.go

```diff
@@ -821,7 +821,14 @@ func (n *Node) ResolveAddress(ctx context.Context, msg *api.ResolveAddressReques
 }
 
 func (n *Node) getLeaderConn() (*grpc.ClientConn, error) {
-	leader := n.Leader()
+	n.stopMu.Lock()
+	if !n.IsMember() {
+		n.stopMu.Unlock()
+		return nil, ErrNoRaftMember
+	}
+
+	leader := n.Node.Status().Lead
+	n.stopMu.Unlock()
```

Comment:
It was n.Leader and it doesn't return error, so I would need to check if it's zero, which is even less obvious. 
Yes, Leader should return error, but it used too widely, so I'd prefer to do it in another PR.